src/components/staking/LiValidator.vue

```diff
@@ -36,7 +36,7 @@
       />
       <div class="validator-info">
         <h3 class="li-validator-name">
-          {{ validator.name }}
+          {{ validator.shortenName }}
```

Comment:
This is a good opportunity to use a custom Vue filter. It can be used like this for example:

`{{ validator.name | truncate(30) }}`

https://v1.vuejs.org/guide/custom-filter.html

Reply:
Thank you! I should have supposed there was something like this at hand

Reply:
I think this truncate will have to work responsively... It is going to be better